discord/audit_logs.py

```diff
@@ -315,7 +315,7 @@ class _AuditLogProxyStageInstanceAction:
 class AuditLogEntry(Hashable):
     r"""Represents an Audit Log entry.
 
-    You retrieve these via :meth:`Guild.audit_logs`.
+    You retrieve these through :meth:`Guild.audit_logs`.
```

Comment:
```suggestion
    You retrieve these via :meth:`Guild.audit_logs`.
```
I prefer the original here

Reply:
Via looks wrong if we want to stay professional

Reply:
Through would be a better way for people who don’t fully understand English